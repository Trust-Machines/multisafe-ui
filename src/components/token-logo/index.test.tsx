import {render} from '@testing-library/react';

import TokenLogo from './index';

test('1 Default render', () => {
    const rendered = render(<TokenLogo address="0x234234234234" />);
    expect(rendered.container).toMatchSnapshot()
});

