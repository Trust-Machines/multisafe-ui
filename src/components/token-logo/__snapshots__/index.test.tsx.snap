// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Default render 1`] = `
<div>
  <img
    class="MuiBox-root css-9uy14h"
    src="/tokens/0x234234234234.svg"
  />
</div>
`;
